Index: deal/src/main/java/ru/gudoshnikova/deal/DealApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package ru.gudoshnikova.deal;\r\n\r\nimport org.springframework.boot.SpringApplication;\r\nimport org.springframework.boot.autoconfigure.EnableAutoConfiguration;\r\nimport org.springframework.boot.autoconfigure.SpringBootApplication;\r\n\r\n@SpringBootApplication\r\npublic class DealApplication {\r\n\r\n    public static void main(String[] args) {\r\n        SpringApplication.run(DealApplication.class, args);\r\n    }\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/deal/src/main/java/ru/gudoshnikova/deal/DealApplication.java b/deal/src/main/java/ru/gudoshnikova/deal/DealApplication.java
--- a/deal/src/main/java/ru/gudoshnikova/deal/DealApplication.java	(revision 7208bb75db39e87fb500f5c92c62def5286e696d)
+++ b/deal/src/main/java/ru/gudoshnikova/deal/DealApplication.java	(date 1774417592137)
@@ -1,7 +1,6 @@
 package ru.gudoshnikova.deal;
 
 import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
